src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ContentDbContext.cs

```diff
@@ -44,6 +44,7 @@ private void Configure(bool updateTimestamps = true)
 
         public DbSet<Methodology> Methodologies { get; set; }
         public DbSet<MethodologyVersion> MethodologyVersions { get; set; }
+        public DbSet<MethodologyVersionContent> MethodologyContent { get; set; }
```

Comment:
Overall seems like a great change to alleviate the current performance problems, combined with the `AsSplitQuery` change on this PR and useful to see them both in action. It will be interesting to see how we can use both approaches elsewhere in future to improve performance.

I agree with your comment about the future work splitting the columns out to separate database tables like we do for Release content. EES-2168 exists for this and you might be interested in taking a look at https://github.com/dfe-analytical-services/explore-education-statistics/compare/dev...EES-2168?diff=unified where I  introduced `MethodologyContentSection` and `MethodologyContentBlock`. As I was also trying to refactor Release content and data blocks at the same time as methodology it became a bit unwieldy and was eventually parked.

If we decide to refactor just Methodology to begin with I'm sure it would be easier to deliver.

I think it would be good to keep the code for  Methodology content and Release content in sync where possible to make maintenance a bit easier and it might make implementing new features like comments and content locking easier too.